mono/metadata/icall.c

```diff
@@ -817,6 +817,61 @@ ves_icall_System_Array_SetValue (MonoArrayHandle arr, MonoObjectHandle value,
 	array_set_value_impl (arr, value, pos, TRUE, TRUE, error);
 }
 
+#ifdef ENABLE_NETCORE
+MonoArrayHandle
+ves_icall_System_Array_InternalCreate (MonoType* type, gint32 rank, gint32* pLengths, gint32* pLowerBounds, MonoError *error)
+{
```

Comment:
Can this really be nohandles ? It returns an array handle.

Reply:
Oops, well, it doesn't compile anyway 😄

Reply:
It can return it through a ref parameter.